dNet/AuthPackets.cpp

```diff
@@ -246,9 +244,9 @@ void AuthPackets::SendLoginResponse(dServer* server, const SystemAddress& sysAdd
 	uint16_t version_major = 1;
 	uint16_t version_current = 10;
 	uint16_t version_minor = 64;
-	GeneralUtils::TryParse<uint16_t>(Game::config->GetValue("version_major"), version_major);
-	GeneralUtils::TryParse<uint16_t>(Game::config->GetValue("version_current"), version_current);
-	GeneralUtils::TryParse<uint16_t>(Game::config->GetValue("version_minor"), version_minor);
+	version_major = GeneralUtils::TryParse<uint16_t>(Game::config->GetValue("version_major")).value_or(version_major);
+	version_current = GeneralUtils::TryParse<uint16_t>(Game::config->GetValue("version_current")).value_or(version_current);
+	version_minor = GeneralUtils::TryParse<uint16_t>(Game::config->GetValue("version_minor")).value_or(version_minor);
```

Comment:
Is there a reason the above declarations arent just simplified to uint16_t version... .value_or(1);?

Reply:
I was debating the readability

Reply:
Went ahead and made the defaults seperate constexpr variables in the next revision for readability, and made the non-defaults seperate. Ideally this is functionally equivalent to the other approach (because they'll be substituted in at compile-time) but more readable.